js_modules/dagster-ui/packages/ui-core/src/assets/AssetEventMetadataEntriesTable.tsx

```diff
@@ -262,11 +262,11 @@ export const AssetEventMetadataEntriesTable = ({
         </AssetEventMetadataScrollContainer>
       ) : null}
       {view === 'plots' ? (
-        <AssetEventMetadataPlots
-          assetKey={assetKey}
-          params={plotView === 'partition' ? {partition: ''} : {time: ''}}
-          assetHasDefinedPartitions={!!assetHasDefinedPartitions}
-        />
+        plotView === 'partition' ? (
+          <AssetTimeMetadataPlots assetKey={assetKey} limit={100} />
```

Comment:
This ternary feels backwards? plotView partition rendering `AssetTimeMetadataPlots`?